src/Wpf.Ui.Gallery/Views/Pages/Text/AutoSuggestBoxPage.xaml

```diff
@@ -4,12 +4,12 @@
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:controls="clr-namespace:Wpf.Ui.Gallery.Controls"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-    xmlns:local="clr-namespace:Wpf.Ui.Gallery.Views.Pages.Text"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     xmlns:system="clr-namespace:System;assembly=System.Runtime"
+    xmlns:text="clr-namespace:Wpf.Ui.Gallery.ViewModels.Pages.Text"
     xmlns:ui="http://schemas.lepo.co/wpfui/2022/xaml"
     Title="AutoSuggestBoxPage"
-    d:DataContext="{d:DesignInstance local:AutoSuggestBoxPage,
+    d:DataContext="{d:DesignInstance text:AutoSuggestBoxViewModel,
```

Comment:
The page itself should be a DataContext, not a view model class.

Reply:
I don't like it, page can have another view model or what ? But I will change.